File: src/targets/php/guzzle/fixtures/multipart-data.php
<?php
<<<<<<< HEAD
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
  'multipart' => [
    [
        'name' => 'foo',
        'filename' => 'src/fixtures/files/hello.txt',
=======

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
  'multipart' => [
    [
        'name' => 'foo',
        'filename' => 'hello.txt',
>>>>>>> upstream/master
        'contents' => 'Hello World',
        'headers' => [
                'Content-Type' => 'text/plain'
        ]
    ],
    [
        'name' => 'bar',
        'contents' => 'Bonjour le monde'
    ]
  ]
]);

echo $response->getBody();

File: src/targets/clojure/clj_http/fixtures/multipart-data.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/post "https://httpbin.org/anything" {:multipart [{:name "foo"
                                                          :content "Hello World"} {:name "bar"
                                                          :content "Bonjour le monde"}]})
=======
(client/post "http://mockbin.com/har" {:multipart [{:name "foo"
                                                    :content "Hello World"} {:name "bar"
                                                    :content "Bonjour le monde"}]})
>>>>>>> upstream/master
